liquid_node/configuration.py

```diff
@@ -203,6 +203,9 @@ def __init__(self):
         self.snoop_pdf_preview_count = self.ini.getint('snoop', 'pdf_preview_count', fallback=1)
         self.snoop_pdf_preview_memory_limit = self.ini.getint('snoop', 'pdf_preview_memory_limit',
                                                               fallback=900)
+        self.snoop_thumbnails_enabled = self.ini.getboolean('snoop', 'thumbnails_enabled', fallback=False)
```

Comment:
let's have these 3 variables have the same prefix `snoop_thumbnails_*` or `snoop_thumbnail_generator_*`

I'd like the prefix to be the same as the service group/task/image/fabio routes, see comment below